packages/jsii-spec/lib/spec.ts

```diff
@@ -432,6 +456,11 @@ export interface Property extends Documentable, Overridable {
      * Implies `static` and `immutable`.
      */
     const?: boolean;
+
+    /**
+     * Where in the module this definition was found
+     */
+    moduleLocation?: SourceLocation;
```

Comment:
`sourceLocation?: SourceLocation`.

Also, since this is present on a number of things, why not introduce a `SourceLocatable` interface and extend that?

Reply:
I guess. Doesn't seem so valuable but I can do that.

Reply:
Well for one, if you want to rename those... There's only one place... And it allows stuff that uses that to be less tied to the list of specific types that have the `sourceLocation` property.